src/MercadoLibre/Provider.php

```diff
@@ -57,9 +36,30 @@ public static function additionalConfigKeys()
      */
     protected function getAuthUrl($state)
     {
-        $url = self::DOMAIN[$this->getConfig('country')] ?? 'https://auth.mercadolibre.com.ar';
+        $domains = [
```

Comment:
is there a reason for moving this out of the const?
Isn't this the same end result

Reply:
Not really, I revered that